examples/archive.html

```diff
@@ -94,7 +94,29 @@ <h3 id="offcanvasRightLabel">Images</h3>
     
     form.addEventListener('submit', (event) => {
       event.preventDefault();
-      const url = input.value.replace('details', 'metadata');
+      let getUrl;
```

Comment:
This is looking good! I'd like to test it out a bit. But can you put it within its own named function, like `function showImages(getUrl) {` below? Maybe it could be named `function extractKey()`?